server/src/chatroomUsers.ts

```diff
@@ -0,0 +1,118 @@
+import express, { query } from "express";
+import { IChatRoom, IPinnedItem, error } from "./types";
+import db from "./firebase";
+import { IBackgroundState } from "./types";
+
+const collection = db.collection("chatroomUsers");
+
+const chatroomUserRouter = express.Router();
+
+//get all chatroom users
+chatroomUserRouter.get("/get-all", async (req, res) => {
+    const doc = await collection.get();
+
+    res.status(200).send(doc.docs.map(doc => doc.data()));
+})
```

Comment:
probably shouldnt include this unless we need it, potentially big read operation and also potential security issue in the future if users can access other user's information